1226-editor/publishings_rights.feature

```diff
@@ -15,13 +15,13 @@ Feature: Editor decides who gets publishing rights
 		| Technical administrative employees |
 	Then the User is a Registrator
 
-	@needJiraTag 
+	@needJiraTagApproval
 	Scenario: Default publishing rights
 	When the Editor of an Institution hasn’t chosen a policy
 	Then the publications policy is:
 		| Registrator has full publishing rights |
 
-	@needJiraTag 
+	@needJiraTagApproval
 	Scenario: Editor defines publishing rights
 	Given a Editor views the Editor page
 	When the Editor chooses one of the three options:
```

Comment:
Skal det ikke være eget steg for å lagre endringene? Kanskje legge til steg med at brukeren får bekreftelse på at endringene er lagret i så fall?

Reply:
And the Editor is notified that a new policy is activated